src/split-pane/index.ts

```diff
@@ -133,8 +131,8 @@ export class SplitPaneBase<P extends SplitPaneProperties = SplitPaneProperties>
 		return content ? [ content ] : [];
 	}
 
-	private _shouldCollapse(dimensions: ContentRect) {
-		const { onCollapse, direction, collapseWidth } = this.properties;
+	private _shouldCollapse(dimensions: ContentRect, collapseWidth: number) {
+		const { onCollapse, direction } = this.properties;
```

Comment:
Why did this get added?

Reply:
because this is one of the examples where moving the properties suddenly broke the typings. Previously collapseWidth (although optional) wasn't causing any issues here. This function is only called from one place and is only called when `collapseWidth` exists, so it seemed sensible to do it like this. Otherwise id have had to do the check in two places.